make/modules/java.base/lib/CoreLibraries.gmk

```diff
@@ -209,3 +209,21 @@ ifeq ($(call isTargetOs, aix), true)
   TARGETS += $(BUILD_LIBJLI_STATIC)
 
 endif
+
+##########################################################################################
```

Comment:
Add a # SapMachine annotation, instead of the ### line

Reply:
Done